PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -2321,13 +2351,12 @@ struct HfTrackIndexSkimCreatorCascades {
   double mass2K0sP{0.}; // WHY HERE?
 
   using SelectedCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::HfSelCollision>>;
-  using TracksWithPVRefitAndDCA = soa::Join<aod::BigTracks, aod::TracksDCA, aod::HfPvRefitTrack>;
+  using TracksWithDCA = soa::Join<aod::BigTracks, aod::TracksDCA>;
 
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0);
   Filter filterSelectTrackIds = (aod::hf_sel_track::isSelProng >= 4);
   // Partition<MyTracks> TracksWithPVRefitAndDCA = aod::hf_sel_track::isSelProng >= 4;
 
-  Preslice<TracksWithPVRefitAndDCA> tracksPerCollision = aod::track::collisionId; // needed for PV refit
```

Comment:
Did this conflict with `trackIndicesPerCollision`?

Reply:
This `Preslice` was not used in the `process` function since no PV refit is performed in the structes that involve V0s and canscades (likely it was a copy paste from the `process2And3Prongs` function of the `HfTrackIndexSkimsCtreator` struct)